corehq/motech/repeaters/models.py

```diff
@@ -258,8 +295,12 @@ def reset_next_attempt(self):
             self.next_attempt_at = None
             self.save()
 
+    def get_repeater_type(self):
+        name = self.__class__.__name__
+        return name[3:] if name.startswith('SQL') else name
+
 
-class SQLCaseRepeater(SyncSQLToCouchMixin, SQLRepeater):
+class SQLCaseRepeater(RepeaterSuperProxy, SyncSQLToCouchMixin, SQLRepeater):
```

Comment:
Is a diamond inheritance pattern typical/recommended for proxy models?

In this case both `RepeaterSuperProxy` and `SQLRepeater` inherit from `models.Model`. Would it make sense to make `SQLRepeater` inherit from `RepeaterSuperProxy` rather than `models.Model`?

Reply:
That actually makes sense, I think it would be a much cleaner way. I will test it out and see how it goes.

Reply:
This is done in https://github.com/dimagi/commcare-hq/pull/30755